src/api/main.py

```diff
@@ -0,0 +1,65 @@
+import os
+from os.path import join
+import errno
+import shutil
+from subprocess import call
+import json
+import uuid
+
+from flask import Flask, jsonify
+
+from rwd.Rapid_Watershed_Delineation import Point_Watershed_Function
+
+app = Flask(__name__)
+
+
+@app.route('/rwd/<lat>/<lon>', methods=['GET'])
+def run_rwd(lat, lon):
+    """
+    Runs RWD for lat/lon and returns the GeoJSON
+    for the computed watershed boundary.
+    """
+    # TODO This function needs to be made more robust to failures.
+
+    data_path = '/opt/rwd-data'
+
+    # Create a temporary directory to hold the output
+    # of RWD
+    output_path = join('/tmp/rwd/', str(uuid.uuid4()))
+    if os.path.exists(output_path):
+        shutil.rmtree(output_path)
+    os.makedirs(output_path)
+
+    Point_Watershed_Function(
+        lat,
+        lon,
+        '1',
+        '10000',
+        data_path,
+        'Delaware_Ocean_stream_dissolve',
+        'delaware_gw_5000_diss',
+        'Delaware_Missing_Coast_Watershed',
+        'DelDEMGeo2fel.tif',
+        'DelDEMGeo2Max_elv_upslope.tif',
+        'DelDEMGeo2ad8_slope_weighted.tif',
+        'DelDEMGeo2ad8.tif',
+        'DelDEMGeo2plen.tif',
+        'DelDEMGeo2tlen_peuker.tif',
+        'DelDEMGeo2gord_peuker.tif',
+        '1',
```

Comment:
np is the number of processes used in the parallel execution of this program.  I agree it should be a constant so you can tweak it.